h/accounts/test/views_test.py

```diff
@@ -259,7 +260,7 @@ def test_login_ajax_converts_non_string_usernames_to_strings(_):
     for input_, expected_output in ((None, ''), (23, '23'), (True, 'True')):
         request = DummyRequest(
             json_body={'username': input_, 'password': 'pass'},
-            auth_domain='hypothes.is')
+            auth_domain='hypothes.is', authenticated_user=None)
```

Comment:
It's probably better to add a fixture which mocks `session.model` and use that consistently across all the `AjaxAuthController` tests. At the moment the implementation details of `session.model` are leaking into tests which are supposed to be testing entirely unrelated things.